packages/es-components/src/components/controls/answer-group/AnswerButton.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { withTheme } from 'styled-components';
+
+const AnswerLabel = styled.label`
+  margin: 0px;
+  background-color: #00a0d2;
+  border: 1px solid #000;
+  flex-grow: 1;
+
+  background-color: ${props => props.variant.bgColor};
+  border-color: transparent;
+  box-shadow: 0 4px 0 0 ${props => props.variant.boxShadowColor};
+  color: ${props => props.variant.textColor};
+  margin-bottom: 4px;
+  margin-top: 0;
+  transition: background-color 250ms linear;
+
+  &[disabled]:hover {
+    background-color: ${props => props.variant.bgColor};
+  }
+
+  &:hover {
+    background-color: ${props => props.variant.hoverBgColor};
+  }
+
+  &:active {
+    margin-bottom: 0;
+    margin-top: 4px;
+    box-shadow: 0 0 0 0 transparent;
+  }
+
+  @media (min-width: ${props => props.theme.screenSize.desktop}) {
+    min-width: ${props => props.itemWidth};
+  }
+`;
+
+const OutlineAnswerLabel = styled(AnswerLabel)`
+  box-shadow: none;
+  background-color: ${props => props.variant.bgColor};
+  border: 2px solid ${props => props.variant.borderColor};
+  border-radius: 5px;
+  color: ${props => props.variant.textColor};
+  font-weight: bold;
+  transition: background-color 250ms linear, color 250ms linear;
+
+  &[disabled]:hover {
+    background-color: ${props => props.variant.bgColor};
+  }
+
+  &:hover {
+    background-color: ${props => props.variant.hoverBgColor};
+    color: ${props => props.variant.activeTextColor};
+  }
+
+  &:active {
+    margin-bottom: 0;
+    margin-top: 4px;
+    box-shadow: 0 0 0 0 transparent;
+  }
+`;
+
+const AnswerDisplay = styled.span`
+  text-align: center;
+  font-size: 16px;
+  font-weight: bold;
+  display: block;
+  padding: 10px 10px 10px 10px;
+  -webkit-touch-callout: none; /* iOS Safari */
+  -webkit-user-select: none; /* Safari */
+  -khtml-user-select: none; /* Konqueror HTML */
+  -moz-user-select: none; /* Firefox */
+  -ms-user-select: none; /* Internet Explorer/Edge */
+  user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
```

Comment:
We're probably okay without the comments on these rules.

Reply:
Removed